css/style.css

```diff
@@ -0,0 +1,166 @@
+body {
+  width: 375px;
+  margin: 0;
+}
+
+header {
+  background-color: #3c3a39;
+}
+
+nav {
+  height: 48px;
+  width: 375px;
+  display: flex;
+  align-items: center;
+  left: 0;
+  top: 44px;
+  border-radius: 0;
+}
+
+#logo-placeholder {
+  display: flex;
+  align-items: center;
+  position: absolute;
+  left: 12px;
+  height: 44px;
+  width: 101px;
+}
+
+h2 {
+  height: 24px;
+  width: 77px;
+  left: 12px;
+  top: 10px;
+  border-radius: nullpx;
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding: 10px 12px 10px 12px;
+}
+
+#normal-button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+  height: 40px;
+  width: 40px;
+  left: 311px;
+}
+
+#menu-button {
+  width: 24px;
+  height: 24px;
+  left: 8px;
+  top: 8px;
+  border-radius: 0;
+  display: flex;
+  order: 0;
+  flex-grow: 0;
+  padding: 8px;
+}
+
+#headline {
+  background-color: #1c1a19;
+  height: 590px;
+  width: 375px;
+  left: 0;
+  top: 0;
+  border-radius: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: relative;
+}
+
+#primary-text {
+  left: 24px;
+  top: 120px;
+  border-radius: nullpx;
+  width: 327px;
+  height: 124px;
+  margin: 0;
+  position: absolute;
```

Comment:
- [ ] I think that if you use Box model _(i.e your elements, their boundaries, and space both in and outside the target element.)_, you will have your elements flow naturally and no need for `position: absolute`. Kindly read about [CSS Box Model here.](https://blog.hubspot.com/website/css-box-model).